plugin.py

```diff
@@ -39,19 +37,26 @@
          "MoonPhases5FM": "MoonPhases5FM.zip",
          "MoonPhases6WG": "MoonPhases6WG.zip",
          "MoonPhases7LQ": "MoonPhases7LQ.zip",
-         "MoonPhases8WC": "MoonPhases8WC.zip"}
-
-
-
+         "MoonPhases8WC": "MoonPhases8WC.zip",
+         "SH_MoonPhases1NM": "SH_MoonPhases1NM.zip",
+         "SH_MoonPhases2WC": "SH_MoonPhases2WC.zip",
+         "SH_MoonPhases3FQ": "SH_MoonPhases3FQ.zip",
+         "SH_MoonPhases4WG": "SH_MoonPhases4WG.zip",
+         "SH_MoonPhases5FM": "SH_MoonPhases5FM.zip",
+         "SH_MoonPhases6WG": "SH_MoonPhases6WG.zip",
+         "SH_MoonPhases7LQ": "SH_MoonPhases7LQ.zip",
+         "SH_MoonPhases8WC": "SH_MoonPhases8WC.zip"}
 
 
 class BasePlugin:
-  
+
     def __init__(self):
         self.debug = False
         self.nextupdate = datetime.now()
-        self.pollinterval = 60  # default polling interval in minutes
+        self.pollinterval = 1440  # default polling interval in minutes
```

Comment:
I think that  default pollinterval should remain in 60 because otherwise, Domoticz, will start to complain about stale hardware. This check interval can be handled inside 'heartbeat' section. I 've done that on PP-Manager.